examples/factors/README.md

```diff
@@ -2,21 +2,202 @@
 
 The _factors_ example is a minimalistic RISC Zero zkVM proof. The prover demonstrates that they know two nontrivial factors (i.e. both greater than 1) of a number, without revealing what those factors are. Thus, the prover demonstrates that a number is composite -- and that they know the factors -- without revealing any further information about the number.
 
-This example was chosen because it is very straightforward. Implementing the verified multiplication and reporting the result in the receipt involves no complexity beyond what is necessary to run the zkVM at all. We therefore hope this example is a good place to look to see all the steps necessary to use the RISC Zero zkVM without any superfluous problem-specific details.
+To see this example in action, [install Rust](install-rust) and use `cargo run` in this directory to run it:
+```
+cargo run
+```
 
-Choosing a simple example necessarily excludes all of the more complex use cases -- so if you are looking for anything beyond the basics, we recommend looking at other examples in this repository!
+# Tutorial
+## How to Recreate _Factors_
 
-## Run this example
+This example was chosen because it is straightforward and is a good introduction for beginners. If you're looking to get started creating RISC Zero zkVM projects, you're in the right place! We'll spend the rest of this readme walking you through how to recreate the factors example for yourself, assuming no prior knowledge of RISC Zero.
 
-First, make sure [rustup](https://rustup.rs) is installed. This project uses a [nightly](https://doc.rust-lang.org/book/appendix-07-nightly-rust.html) version of [Rust](https://doc.rust-lang.org/book/ch01-01-installation.html). The [`rust-toolchain`](rust-toolchain) file will be used by `cargo` to automatically install the correct version.
+## Step 1: Create a new project
+
+First, [install rust](install-rust) if you don't already have it. Next you can create a RISC Zero zkVM project with boilerplate already filled out using our [`cargo risczero` tool](cargo-risczero):
```

Comment:
both these links are broken afaict

Reply:
Hmm, I wonder what's different from the working links in https://github.com/risc0/risc0/blob/main/README.md... I'll have to investigate further

Reply:
links there use [...][...] whereas here you use [...](...)

Reply:
though i think the syntax victor used here is best: https://github.com/risc0/risc0/edit/main/examples/waldo/README.md